src/main/java/at/hannibal2/skyhanni/features/garden/FarmingFortuneDisplay.kt

```diff
@@ -345,9 +347,9 @@ object FarmingFortuneDisplay {
 
         for (line in tool?.getLore()!!) {
             tooltipFortunePattern.matchMatcher(line) {
-                displayedFortune = group(1)!!.toDouble()
-                reforgeFortune = group(2)?.toDouble() ?: 0.0
-                gemstoneFortune = group(3)?.toDouble() ?: 0.0
+                displayedFortune = group("display")!!.toDouble()
```

Comment:
```suggestion
                displayedFortune = group("display")?.toDouble() ?: 0.0
```

Reply:
does this work without exploding? i thought you had to use `toDoubleOrNull()`